app.py

```diff
@@ -377,8 +377,17 @@ async def on_message(message):
                 for error in spilt_discord_message(ret.get("error")):
                     await thread.send(error)
                     logger.info(error)
-            await thread.send("Order SUCCESSFUL ✅ \n")
-            thread_message = f'🟢 {cur_date} -- {order.get("coin_pair")} {order.get("coin_pair")}'
+                    
+                if ret.get("status") and ret.get("status") == 400:
+                    await thread.send("Order Skipped ❌ \n")
```

Comment:
You forgot the detail message that i mentioned to you last time

Reply:
Add reason details at behind of message (16 Sept)